packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1322,16 +1329,12 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Handles dismissing (cancelling) the menu
    */
   private _onDismiss = (): void => {
-    // reset the selected index
-    // to the last valud state
-    this._resetSelectedIndex();
-
     // close the menu and focus the input
-    this.setState({ isOpen: false });
+    this._setOpenStateAndFocusOnClose(false, false);
 
-    if (this._autofill.current && this._focusInputAfterClose) {
-      this._autofill.current.focus();
-    }
```

Comment:
Are there any cases when we want to focus the input after close? Now you are always sending false here

Reply:
onDismiss is only ever triggered when we click out via a click event. Any other event was caught by another event handler. 

So with all that in mind, there shouldn't be any scenario that should put focus in the autofill when we click out of it.